kernel/src/main/java/org/kframework/backend/kore/ModuleToKORE.java

```diff
@@ -598,8 +610,14 @@ private void convertRule(Rule rule, boolean heatCoolEq, Sort topCellSort, Map<St
                     rule = rule.withAtt(rule.att().add(Attribute.SIMPLIFICATION_KEY));
                 }
             }
+            sb.append("  axiom{R");
+            Option<Set> sortParams = rule.att().getOption("sortParams", Set.class);
+            if (sortParams.nonEmpty()) {
+                for (Object sort : sortParams.get())
+                    sb.append("," + sort);
```

Comment:
I'd rather we cast this to a Sort and called name()

Reply:
Actually `sortParams` is a set of `String`s  holding the sort parameter names, so no casting is needed. I changed the name of the iterator to make that clear.